en/other/faq/technical.md

```diff
@@ -1399,6 +1391,17 @@ On the card payment method’s [callback](/en/core-api/credit-card.md#_3ds-authe
 
 However, most of the time from a merchant perspective, you don’t need to know whether the transaction was processed via 3DS 2.0 or not, or if the customer was prompted with the 3DS challenge-input or not. As a merchant what usually you need to know [is just the ECI code](https://support.midtrans.com/hc/en-us/articles/204161150-What-is-ECI-on-3DS-protocol-). To determine whether the transaction was 3DS or not. If the transaction was 3DS most of the time, the risk liability (in case of fraud, etc.) does not fall to the merchant.
 
+#### How to test offline installment on Sandbox environment?
+Offline Installment is the type of payment where Card Issuing Bank used for making an installment payment and the Acquiring Bank need not be the same. For example, a customer makes an installment payment using BNI Card and the Acquiring Bank is Mandiri.
+
+You can use for example `"481111, 521111"` as dummy BINs value, which have corresponding test cards that are acceptable on Sandbox environment. You can check [here for more sandbox test cards](/en/technical-reference/sandbox-test.md#card-payments) that you can use as BINs value. E.g. choose some cards, and then use their first 6-digit as BINs value.  
+
+Please only use the dummy BINs value as testing purpose on Sandbox, so that you can test to the point that the transaction is successfully accepted as expected on Sandbox.
+When you are in a Production environment you need to change the BINs value to real BINs card issuer.
+Also you can use real BINs value on Sandbox, but please remember do not pay the transaction on Sandbox. You just need to test the BINs value to proceed and show the 3DS/OTP page.
```

Comment:
What do you expect this 3 lines to be rendered? Here's how markdown will be rendered:

If you expect the 3 lines to have blank line between them, use 2 `enter` char:
```
Line 1.

Line 2.

Line 3.
```

If you expect the 3 lines to render as paragraph, no need to use any `enter` char:
```
Line 1. Line 2. Line 3.
```

Your code uses 1 `enter` char, which is not consistent to how we usually format this docs. Please choose one of above.

Reply:
> Also you can use real BINs value on Sandbox, but please remember do not pay the transaction on Sandbox. You just need to test the BINs value to proceed and show the 3DS/OTP page.

Better wording:
> Alternatively, you can use real BINs value on Sandbox, you only will be able to test & proceed up until the 3DS/OTP page. That should be enough to test & ensure that the BINs param is valid & working. But please expect the payment will be denied after 3DS/OTP, because the card likely is not recognized by our Sandbox.